source/javascripts/stages/stage_2.rb

```diff
@@ -26,7 +26,7 @@ def code
     def instructions
       <<-INSTR
 <p>Well done!</p>
-<p>Now please make the code above work by implementing a method called `upper` that will return a given string converted to all capitals</p>
+<p>Now please make the code above work by implementing a method called `upper`. This method should accept an argument and convert the string value to be in all capitals.</p>
```

Comment:
Are you sure you want to convert the original string? The wording is a little confusing - maybe use the original wording, but also specifying the requirement for an argument?